charts/app-config/templates/signon-secrets-sync-configmap.yaml

```diff
@@ -21,6 +22,7 @@ data:
       "Imminence",
       "Link Checker API",
       "Local Links Manager",
+      "Locations API",
```

Comment:
There doesn't seem to be a `Locations API` app [in Signon](https://signon.publishing.service.gov.uk/doorkeeper_applications).

Reply:
thanks - for the spot on this Chris